src/main.c

```diff
@@ -57,13 +57,15 @@ struct netdata_static_thread static_threads[] = {
 #elif defined(__APPLE__)
     // macOS internal plugins
     {"PLUGIN[macos]",        CONFIG_SECTION_PLUGINS,  "macos",      1, NULL, NULL, macos_main},
+#elif defined(__OpenBSD__)
+    // OpenBSD internal plugins
```

Comment:
Will populate this later, after I dig through the FreeBSD plugin and see what needs changing.